virttest/qemu_storage.py

```diff
@@ -345,10 +346,10 @@ def snapshot_del(self, blkdebug_cfg=""):
                                        " parameters")
         if blkdebug_cfg:
             cmd += " blkdebug:%s:%s" % (blkdebug_cfg, self.image_filename)
+            return process.run(cmd, ignore_status=True)
```

Comment:
You set ignore_status=True, which makes process.run() *won't* raise Exception any more.
I know it's your intention.